packages/react-core/src/evm/hooks/useENS.ts

```diff
@@ -26,19 +28,29 @@ export function useENS() {
         return null;
       }
 
-      const provider = getChainProvider(1, {
-        clientId,
-        supportedChains: ethereum
-          ? [
-              {
-                chainId: 1,
-                rpc: [...ethereum.rpc],
-                nativeCurrency: ethereum.nativeCurrency,
-                slug: ethereum.slug,
-              },
-            ]
-          : undefined,
-      });
+      let provider = null;
+      if (chainId == 8453) {
+        // if the active chain is Base, then establish a provider using Base Name Service to resolve addresses and avatars
+        provider = new providers.JsonRpcProvider("https://mainnet.base.org/", {
```

Comment:
We'd need to use our own RPC here. Can you add an optional ensAdress param to `getChainProvider` instead? So it all goes through the same flow

Reply:
Hi @joaquim-verges, thanks for the quick review and I am looking into using ```getChainProvider``` for the integration.